internal/mode/static/state/conditions/conditions.go

```diff
@@ -62,7 +62,7 @@ const (
 	//
 	// FIXME(bjee19): Update to Gateway sig v1 version when released.
 	// https://github.com/nginxinc/nginx-gateway-fabric/issues/1168
-	RouteConditionPartiallyInvalid v1beta1.RouteConditionType = "PartiallyInvalid"
+	RouteConditionPartiallyInvalid v1.RouteConditionType = "PartiallyInvalid"
```

Comment:
Could we also update to  https://github.com/kubernetes-sigs/gateway-api/blob/adbd9968d4ff7995af21cb9cd0913288e2b4ca27/apis/v1/shared_types.go#L435  and remove the FIXME?